consistence/nsq_lookupd_etcd.go

```diff
@@ -416,6 +449,17 @@ func (self *NsqLookupdEtcdMgr) scanTopics() ([]TopicPartitionMetaInfo, error) {
 	self.topicMetaInfos = topicMetaInfos
 	self.tmisMutex.Unlock()
 
+	//update topic meta info
+	topicMetaMapRef := make(map[string]*TopicMetaInfo)
+	for topic, _ := range topicMetaMap {
```

Comment:
Maybe change `topicMetaMap` to `map[string]*TopicMetaInfo` to avoid copy again. And change the `self.topicMetaInfos` and `self.topicMetaMap` in same lock